go/tools/bzltestutil/init.go

```diff
@@ -30,7 +30,11 @@ var (
 	testExecDir string
 )
 
-// This initializer runs before any user packages.
+// Before Go 1.21, this initializer runs before any user packages.
+//
+// Since Go 1.21, the order of `init()` calls is determined based
```

Comment:
Based on my understanding of the process, the sorting isn't determined by the runtime but fixed by the spec. Instead of trying to describe it here, we could just link to the authoritative source.

Reply:
fixed